src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  background-color: #fafafa;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+main {
+  box-shadow: 0 4px 8px 0 rgba(0 0 0 / 20%);
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10%;
+  max-width: 375px;
+}
+
+hr {
+  width: 99.9%;
+  border: 0.05px solid #edeaea;
+  color: #edeaea;
+  margin-bottom: 0;
```

Comment:
To avoid repetition you can easily use
margin : 0 auto
The 0 for margin up and down 
The auto for left and right

Reply:
Thanks remove all repeated margin parameter accordingly.